test/core/hash_algorithm_signature_body_signer_tests.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:convert';
+import 'dart:typed_data';
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
Why do we need flutter_test? This should be a dart-only package since it doesn't have any UI in it. I assume the test framework should not reference Flutter at all.

Reply:
correct, a commit containing those changes is added, now using test: instead of flutter_test: